pipelines/common/src/main/java/com/google/fhir/analytics/DwhFiles.java

```diff
@@ -70,6 +79,33 @@ private DwhFiles(String dwhRoot) {
     this(dwhRoot, FhirContexts.forR4());
   }
 
+  /**
+   * In order to interpret the file paths correctly the {@code dwhRoot} has to be represented in one
+   * of the below formats according to the filesystem platform:
+   *
+   * <p>Linux/Mac:
+   *
+   * <ul>
+   *   <li>TestRoot/SamplePrefix
+   *   <li>/Users/beam/TestRoot/SamplePrefix
+   * </ul>
+   *
+   * <p>Windows OS:
+   *
+   * <ul>
+   *   <li>TestRoot\\SamplePrefix
+   *   <li>C:\\Users\\beam\\TestRoot\\SamplePrefix
```

Comment:
nit: If you are using double `\` to compensate for the java escape character, please add `"` around these strings to make it more clear OR use single `\`.

Reply:
Ok, changed to single backslash `\`.